app/models/miq_provision_virt_workflow.rb

```diff
@@ -119,8 +119,7 @@ def set_on_vm_id_changed
        :placement_rp_name,
        :linked_clone,
        :snapshot,
-       :placement_dc_name,
-       :placement_storage_profile]
```

Comment:
`clear_field_values` is resetting `:placement_storage_profile` to `[nil, nil]` which will signal to `allowed_storage_profile` that `:placement_storage_profile` is already initialized and so `allowed_storage_profile` will not load it with the `src[:vm].storage_profile`.

During the page loading process, this ` set_on_vm_id_changed` is being called. The `allowed_storage_profile` is also called multiple times and will act correctly the first time. But then won't because the `:placement_storage_profile` becomes `[nil, nil]` (instead of `nil`).

(Clarification: so my first fix solved the original reported problem but then broke the provision workflow. Removal of this entry here will solve my new problem)